### ISSUE TITLE
Changes Changelog window for be more practical

### ISSUE THREAD

**Issue by Fireztonez** [gorilla-devs/GDLauncher#805]
**Is your feature request related to a problem? Please describe.**
Presently the Changelog window is really small and not really practical to use. It is sincerly a pain to look big changelog with that UI... A small look and I think you will understand.

When you click, question to saw the latest update changelog, you click on the little button and you have a beautiful empty page:
![image](https://user-images.githubusercontent.com/19277301/110407724-5d8b0e80-8052-11eb-8871-f2e58c58307a.png)

After you have to click once on the menu and click on the latest version. And now you have that:
![image](https://user-images.githubusercontent.com/19277301/110407683-477d4e00-8052-11eb-8a79-8a0927cd91d1.png)

Basically, when I see a new mod update, I have basically to click 4 times (Click to open the Mod Window, click on changelog, click to open the list of version and click on the version you want) question to have a look to the latest changelog. A first thing I think you should do, question to make it a lot more practicle, is simply to automatically show the latest changelog when you open the changelog window. (If you want to see other one you still can change the version on the menu at the top.).

Second issue, the Font-Size is really big, and the windows is really minuscule (And like all of Popup window in your application, it is impossible to resize). I think it will be a lot more practicle to see that on a bigger window and with a more resonable font-size.

I just override 3 litlle value on the CSS question to show an example:
![image](https://user-images.githubusercontent.com/19277301/110409477-587b8e80-8055-11eb-863e-cf4346401ba3.png)
*Changelog font-size: 14px (Compare to 20px before)
  Window height: 700px;
  Window width: 950px;*

**Comment by Fireztonez** [comment]
Other little problem we saw, and again this can be fixed really easy on the CSS, in the changelog text, when the line is too short, the word is cut and show the rest on a new line. You should probably cut the line at the end of a word and go to a new line for the next one.

Question to be more clear, you can see a really really frequent thing with your changelog right now (Due to the window size and Font-size):
![image](https://user-images.githubusercontent.com/19277301/110411618-dd1bdc00-8058-11eb-9ee3-2f031461c6e9.png)
*You can see the word *"reflects"* is cut and show *"refl ects"*

I fixed that rapidly by overriding `word-break: break-all;` from the class "jwMDxF" by `word-break: break-word;`

And now see the result:
![image](https://user-images.githubusercontent.com/19277301/110412065-98447500-8059-11eb-8828-1cbc783ec77e.png)

**Comment by blarfoon** [comment]
I agree with everything, can you open a PR to do these things? I will gladly accept that

**Comment by TheDeviantCrafter** [comment]
Two minor suggestions:

1. When checking for updates in the mod list screen, add an [ i ] button that opens the changelog for the most recent version.
2. In the changelog screen, highlight the version of the mod that is currently installed.

These would make changelogs easier to find, and make it easier to tell which ones are relevant.
